vernac/library.ml

```diff
@@ -318,15 +323,19 @@ let native_name_from_filename f =
 
 let register_library m =
   let l = m.library_data in
-  Declaremods.Synterp.register_library
-    m.library_name
-    l.md_syntax_objects;
   Declaremods.Interp.register_library
     m.library_name
     l.md_compiled
     l.md_objects
     m.library_digests
     m.library_extra_univs;
+  register_native_library m.library_name
```

Comment:
Would it make sense to merge register_native_library and

Reply:
It probably makes sense, yes, but there's a (kind of strange) dependency on `library_full_filename`, so I think it would deserve to be studied a bit carefully in a separate patch.